sample/todo-sync/src/com/cloudant/todo/TasksModel.java

```diff
@@ -51,7 +52,7 @@
     private static final String DATASTORE_MANGER_DIR = "data";
```

Comment:
For consistency with the new naming we should probably change `DATASTORE` to `DOCUMENT_STORE` for this and other occurrences of `Datastore` in this file. 

Also, we should probably fix the typo `MANGER`.